themes/README.rst

```diff
@@ -0,0 +1,146 @@
+#####################
+Comprehensive Theming
+#####################
+
+
+Comprehensive Theming lets you customize the appearance of your Open edX
+installation.  You can override Sass and CSS settings, images, or entire HTML
+templates.
+
+Eventually, Comprehensive Theming will obsolete existing theming mechanisms,
+but for now they co-exist peacefully. This document describes how to use 
+Comprehensive Theming, and also the changes you'll need to make to keep other
+theming mechanisms working.
+
+
+Creating a theme
+================
+
+A theme is a directory of assets.  You can create this directory wherever you
+like, it does not have to be inside the edx-platform directory.  The structure
+within this directory mirrors the assets in the edx-platform repo itself.
+Files you provide in your theme are used in place of the same-named files in
+edx-platform.  Here's a sample::
+
+    my-theme
+    └── lms
+        ├── static
+        │   ├── images
+        │   │   └── logo.png
+        │   └── sass
+        │       ├── _overrides.scss
+        │       ├── lms-main-rtl.scss
+        │       └── lms-main.scss
+        └── templates
+            ├── footer.html
+            └── header.html
+
+The top directory is named whatever you like.  This example uses "my-theme".
+The files provided here override the files in edx-platform.  In this case, the
+``my-theme/lms/static/sass/lms-main.scss`` file is used in place of the
+``edx-platform/lms/static/lms-main.scss`` file.
```

Comment:
Should be `edx-platform/lms/static/sass/lms-main.scss` instead of `edx-platform/lms/static/lms-main.scss`.